web/src/portico/landing-page.js

```diff
@@ -177,10 +124,6 @@ const events = function () {
     if (path_parts().includes("apps")) {
```

Comment:
Can we also clean this up similarly?

Reply:
Sure

Reply:
I will open a different PR for it and fix it similarly for teams too.